### DISCUSSION TITLE
Announce For Zero

### DISCUSSION THREAD

**Discussion by IMGROOT2** [For-0/vocabustudy#193] [Q&A] [answered]
Should I wait until 1.0.0 or announce it now? Also, we need to update Omkar's locker number on nightly
![image](https://user-images.githubusercontent.com/116324098/197934712-4cb31a46-8966-43ec-b00d-9084fa0902b5.png)

P.S: Here's the changelog:

![image](https://user-images.githubusercontent.com/116324098/197934874-105032ac-5b52-46c4-b67d-c33382958edd.png)

**Comment by grimsteel** [comment, marked as answer]
That's a pretty good announcement - I think you should mention that ***we*** created For Zero as a result of Vocabustudy - otherwise people will think some random company bought it lol

You can announce it when ready

**Comment by grimsteel** [comment]
That changelog embed looks really good

**Comment by grimsteel** [comment]
> we need to update Omkar's locker number

Create an issue and assign it to him

**Comment by IMGROOT2** [comment]
> That's a pretty good announcement - I think you should mention that ***we*** created For Zero as a result of Vocabustudy - otherwise people will think some random company bought it lol
> 
> 
> 
> You can announce it when ready

Alr ty
